[docs] small fixes in the content for the client cli (#14505)
MystenLabs/sui: docs/content/references/cli/client.mdx

### Get complete object information

> Use `sui client object <OBJECT-ID>` to list an object's information by the ID you provide. In this example a Coin object's information is displayed.

Reviewer: ```suggestion
Use `sui client object <OBJECT-ID>` to list object information for the ID you provide. This example displays the information for a Coin object.
```

Reviewer: Switching back to active voice on that last sentence. And avoid using possessive for inanimate objects. In other words, "to list object information" is purposively suggested instead of "to list an object's information"
